modules/avatar/avatar.go

```diff
@@ -8,44 +8,39 @@ import (
 	"bytes"
 	"fmt"
 	"image"
-	"image/color/palette"
 
 	_ "image/gif"  // for processing gif images
 	_ "image/jpeg" // for processing jpeg images
 	_ "image/png"  // for processing png images
 
+	"code.gitea.io/gitea/modules/avatar/dice_bear"
+	"code.gitea.io/gitea/modules/avatar/identicon"
 	"code.gitea.io/gitea/modules/setting"
-	"code.gitea.io/gitea/modules/util"
 
-	"github.com/issue9/identicon"
 	"github.com/nfnt/resize"
 	"github.com/oliamb/cutter"
 )
 
 // AvatarSize returns avatar's size
 const AvatarSize = 290
 
+type RandomAvatarGenerator interface {
```

Comment:
should we do some sort of interface & register or just import and use sub packages?